src/components/TabulatorComponent.vue

```diff
@@ -23,6 +27,11 @@ export default class TabulatorComponent extends Vue {
 
   private tabulatorInstance: Tabulator | null = null;
 
+  get eventOptions(): Object {
+    const events = eventFactory.bind(this)
+    return {...events(rowEvents), ...events(cellEvents)}
```

Comment:
Import events (row and cell) manually is **expensive** because the developer need know and reminder that.
We can use a webpack feature to handle that: [require context](https://webpack.js.org/guides/dependency-management/#requirecontext).
You can see a real example [here](https://github.com/chrisvfritz/vue-enterprise-boilerplate/blob/master/src/components/_globals.js).

Plese, try provide a test case (maybe using cypress) for the events.